docs/course_materials/exercises/01_Linux/check_class_exercise_1.py

```diff
@@ -0,0 +1,28 @@
+import os.path
+from pathlib import Path
+import check_helper
+
+home = str(Path.home())
+exercise_commands = {'mkdir': 'creating directories', 'tree': 'viewing the directory tree'}
+exercise_paths = [home + '/test/tmp']
+
+def main():
+    '''
+    Running the tests and prompting the student according to his work.
+    '''
+    test_result = ''
+    student_commands = check_helper.get_student_commands()
+    unused_commands = check_helper.find_unused_commands(student_commands, exercise_commands)
+    if len(unused_commands) > 0:
+        test_result += "It seems you missed some commands. Make sure you have used the following commands:\n"
```

Comment:
here I meant it could be great to present exactly which command was missed, or which exactly directory wasn't created.

Reply:
It's doing that for the commands.
For the path, I didn't break it down to pieces. It's possible to add more paths to the check.